tests/test_addons.py

```diff
@@ -1251,7 +1251,8 @@ def configure_addon(self):
         self.user_addon.oauth_grants[self.node._id] = {self.oauth_settings._id: []}
         self.user_addon.save()
 
-    def test_serialize_addons(self):
+    @mock.patch('addons.github.models.NodeSettings.get_folders', return_value=[])
```

Comment:
Were these tests making outgoing requests? 😱

Reply:
Afraid so via the Github client.

Reply:
Oh dear, glad to get this one in then!